src/components/interface/AudioPlay.jsx

```diff
@@ -1,26 +1,78 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+
 import { withStyles } from '@material-ui/core/styles';
 import PlayCircleOutlineIcon from '@material-ui/icons/PlayCircleOutline';
+
 import styles from '../styles';
+import Button from '../interface/Button';
+import * as firebase from 'firebase';
 
 class AudioPlayer extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      disabled: false
+      audioFile: undefined,
+      metadataLoaded: false
     };
   }
 
-  // TODO: handle audio playing file
+  loadAudio = () => {
+    this.pathRef.getDownloadURL().then((url) => {
+      fetch(url).then(() => {
+        this.setState({ audioFile: new Audio(url) });
+        this.state.audioFile.addEventListener('loadeddata', () => {
```

Comment:
It's nice in JS when we don't need to do object references at such a deep level, like `this.state.audiofile....`.

Would it be the same in this case if we did:
```
const audioFile = new Audio(url);
audioFile.addEventListener ....
this.setState({ audioFile }) // <- this actually is equivalent to setting {audioFile: audioFile}
```

Reply:
> const audioFile = new Audio(url); audioFile.addEventListener .... this.setState({ audioFile }) // <- this actually is equivalent to setting {audioFile: audioFile}
also works. I'll use it instead.